apps/admin/core/components/common/controller-input.tsx

```diff
@@ -23,7 +23,7 @@ export type TControllerInputFormField = {
   key: string;
   type: "text" | "password";
   label: string;
-  description?: string | JSX.Element;
+  description?: string | React.ReactNode;
```

Comment:
We don’t need to include `string` everywhere now since `ReactNode` already has them, but we can skip that change right now.